cypress/integration/portal-dashboard/portal-dashboard-activity-score.spec.js

```diff
@@ -0,0 +1,402 @@
+import ActivityScore from "../../support/elements/portal-report/activity-score-settings";
+
+context("Portal Dashboard Anonymous Mode",() =>{
+
+  const score = new ActivityScore();
+
+  beforeEach(() => {
+    cy.visit("/?portal-dashboard");
+    cy.get('[data-cy=navigation-select]').click();
+    cy.get('[data-cy="list-item-feedback-report"]').should('be.visible').click();
+  });
+
+    it('verify activity level feedback is displayed by default',()=>{
+      cy.get('[data-cy=activity-level-feedback-button]').invoke("attr", "class").should("contain", "active");
+    });
+    it('verify activity level feedback content',()=>{
+      cy.get('[data-cy=feedback-note-toggle-button]').invoke("attr", "title").should("contain", "Note on activity-level feedback");
+      cy.get('[data-cy=feedback-settings-toggle]').should("exist");
+      cy.get('[data-cy=feedback-settings-toggle]')
+      .should("contain", "Activity Score:")
```

Comment:
I think that some of the values like '[data-cy=activity-level-feedback-button]' and '[data-cy=feedback-settings-toggle]' that are used 5 or 10 times should either be constants or should be put in methods to be called multiple times. The methods could be here in portal-dashboard-activity-score.spec.js, or perhaps in activity-score-settings.js instead.